DigitalLearningSolutions.Web/ViewModels/Common/ViewComponents/NumericInputViewModel.cs

```diff
@@ -0,0 +1,46 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.Common.ViewComponents
+{
+    namespace DigitalLearningSolutions.Web.ViewModels.Common.ViewComponents

```

Comment:
Looks like the view model here has namespaces doubled up which is causing some funky knock on effects that I'll highlight in other comments. Hopefully getting this back to a sensible namespace will mean the other effects will be easily solvable.

Reply:
Ahhh that's what was going wrong. I was very confused why a bunch of files needed me to change the imports all of a sudden. Fixed this namespace and changed all those imports back. Thanks for catching that!